src/ifp/test/make_tracks1.py

```diff
@@ -0,0 +1,21 @@
+from openroad import Tech, Design
+import helpers
+
+tech = Tech()
+tech.readLEF("Nangate45/Nangate45_tech.lef")
+tech.readLEF("Nangate45/Nangate45_stdcell.lef")
+
+tech.readLiberty("Nangate45/Nangate45_typ.lib")
+
+design = Design(tech)
+design.readVerilog("reg1.v")
+design.link("top")
+
+floorplan = design.getFloorplan()
```

Comment:
Did we consider having a Floorplan object (and one for each step of the flow) the design as a parameters? 

(sorry if that was already discussed before, I'm not sure where the discussion landed on this on the design doc: https://docs.google.com/document/d/e/2PACX-1vSNVgum66oml5bOw41hNSjWuyuBe4wF7S1AXGXPwXOuloRRgchw9dEjPr_ZfQu39FbC_4EK8e12rA8p/pub).

Reply:
What would be the benefit?

Reply:
It could make the lifecycle of those objects a bit more explicit.

With the current client code it's not really clear (at least to me) what the relation between the design and the floorplan and the design object and which one holds the current state.

I think @growly had interesting suggestions on the original doc about this.

Reply:
The database holds the state.  Anything in python is just holding a reference to the db objects.  Lifecycle seems very simple to me - you get the floorplan object, do what you want, and let python reclaim it when done.

Reply:
So the `Design` type primary function is to do the following?
- holds the db
- provide factory method to instanciate all of the flow operation

Reply:
Yes.  It will know how to provide all the necessary args to instantiate the other tools